contribution.md

```diff
@@ -81,11 +82,25 @@ In the created repositories add also robot account which is mandatory for publis
 
 For Fedora case, you need ask for adding robot account cverna@redhat.com, but ask SCLORG team.
 
+## Typical contribution session
+
+You should be informed about each repository package's requirements, as they can slightly differ. Usually, those packages are needed:
+
+- [distgen](https://github.com/devexp-db/distgen/) - is used for generating image source files,
+- [go-md2ma](https://github.com/cpuguy83/go-md2man) - is used for generating documentation from README.md
+
+1. Fork the repository
+2. Run `git submodule update --init` to download the `common` submodule with `common/common.mk` makefile. We use this submodule for common tests and functions across all our repositories.
+3. Add a feature or bug fix in the templates (src directory) or values (specs/multispec.yml file)
+   1. If you do not change anything in those directories, you do not need to regenerate files
```

Comment:
This step is not typical at all - we only use distgen generation in a small number of repositories. I would just go with a generic "Add a feature or bug fix" step. If you want to, you can explicitly state that for distgen-enabled repositories the sources are in the `/src` directory and so on.

Reply:
Fixed in the last commit. Mentioned with `*`.